cdap-app-fabric/src/main/java/io/cdap/cdap/scheduler/ScheduleNotificationSubscriberService.java

```diff
@@ -199,6 +202,12 @@ protected void processNotification(ProgramScheduleStoreDataset scheduleStore,
         scheduleId = ScheduleId.fromString(scheduleIdString);
       }
 
+      if (isApplicationDisabled(scheduleId.getNamespace(), scheduleId.getApplication())) {
+        LOG.debug("Application {}.{} is disabled, ignoring the schedule {}.", scheduleId.getNamespace(),
+                 scheduleId.getApplication(), scheduleId.getSchedule());
```

Comment:
Wrong alignment. The `"` and `s` should be aligned.

Reply:
Done